openedx/features/partners/helpers.py

```diff
@@ -107,3 +109,13 @@ def get_partner_from_user(user):
 
 def user_has_performance_access(user, partner):
     return bool(partner and user.has_perm('partners.%s' % PERFORMANCE_PERM_FRMT.format(slug=partner.slug)))
+
+
+def get_partner_registration_countries(partner_slug):
```

Comment:
Let's please make a generic helper for this one as well instead of coupling it with partner.

Reply:
This stands true if we decide to make top countries same for every partner. Otherwise, we'd need the partner slug, and couple it with partners as it is. Something we can decide after discussing requirements with product team.